src/runner.rs

```diff
@@ -191,13 +194,16 @@ pub mod platform {
     use winreg::enums::*;
 
     pub fn firefox_default_path() -> Option<PathBuf> {
+        if let Some(path) = find_binary("firefox.exe") {
+            return Some(path)
+        }
         let opt_path = firefox_registry_path().unwrap_or(None);
```

Comment:
This feels pretty wrong on Windows. I'm not sure that people would expect `%path%` to take precedence over the registry information about where firefox is installed. On OSX it makes sense since it's taking precedence over some likely hardcoded paths.